shell/platform/darwin/ios/ios_context.mm

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/darwin/ios/ios_context.h"
+
+#include "flutter/fml/logging.h"
+#include "flutter/shell/platform/darwin/ios/ios_context_gl.h"
+
+#if FLUTTER_SHELL_ENABLE_METAL
+#include "flutter/shell/platform/darwin/ios/ios_context_metal.h"
+#endif  // FLUTTER_SHELL_ENABLE_METAL
+
+namespace flutter {
+
+IOSContext::IOSContext() = default;
+
+IOSContext::~IOSContext() = default;
+
+std::unique_ptr<IOSContext> IOSContext::Create(IOSRenderingAPI rendering_api) {
+  switch (rendering_api) {
+    case IOSRenderingAPI::kOpenGLES:
+      return std::make_unique<IOSContextGL>();
+    case IOSRenderingAPI::kSoftware:
+      break;
```

Comment:
In particular right here, we'll fall into `FML_CHECK(false)` when running on a simulator.  Or am I missing something?